src/app/router/ui/RootRoute.tsx

```diff
@@ -1,10 +1,11 @@
 import { Outlet } from 'react-router-dom';
+import { TrendingMovies } from '../../../features/TrendingMovies';
 
 export const RootRoute = () => {
-  return (
-    <>
-      <span className="text-9xl">Hello World</span>
-      <Outlet />
-    </>
-  );
+    return (
+        <>
+            <TrendingMovies />
```

Comment:
Let's stick with the next structure:
- RootRoute is used as generic route for pages or you can call it generic wrapper for pages; That's why let's create a Main page at pages folder and  put TrendingMovies there, because it's a part of a Main page 
For routes use this reference - https://github.com/Aleksey-Timoschenko/production-app/blob/main/src/app/router/router.tsx